intel/BDTK
# [POAE7-2573] refine BDTK doc structure

- BDTK in 10 minutes
    - Introduction 
    - APIs
- User Guide
    - How to use
        - Environment Preparation
        - Run with Prestodb
        - How to run simple examples with Prestodb in DEV environment
        - How to run simple examples with Prestodb in distributed environment
    - Supported features
        - Data Types
        - Aggregate Functions
        - Conditional Expression
        - Conversion Functions
        - Logical and Comparsion ops
        - Mathematical Functions
        - String Functions
        - Time Functions
        - Sql Syntax
    - Utils
        - Sql2IR
- Developer Guide
    - How to contribute
        - How to build
        - How to debug/test
        - Code style check
        - Documentation
        - Troubleshooting
    - Implementation Specification
        - Exception Specification
        - CiderAllocator use guide
        - Logger
- Sharing
    - Live Demo
        - Design Doc
        - Performance Report
    - External Link
    - Paper/ conference/ talk
